shared/wallets/export-secret-key/index.js

```diff
@@ -19,7 +22,11 @@ export default class ExportSecretKeyPopup extends React.Component<Props> {
   render() {
     const header = (
       <React.Fragment>
-        <Text type="BodySmallSemibold">{this.props.username}’s account</Text>
+        {this.props.walletName ? (
```

Comment:
all wallets have a name now right?

Reply:
Yeah, I believe so, just thought I'd be safe, dunno if previously-created wallets are getting names.